src/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects/SqlProjectBuilderExtensions.cs

```diff
@@ -57,6 +57,24 @@ public static IResourceBuilder<SqlProjectResource> AddSqlProject(this IDistribut
                       .ExcludeFromManifest();
     }
 
+    /// <summary>
+    /// Adds a SQL Server Database Project resource to the application based on a referenced NuGet package.
+    /// </summary>
+    /// <typeparam name="TPackage">Type that represents the NuGet package that contains the .dacpac file.</typeparam>
+    /// <param name="builder">An <see cref="IDistributedApplicationBuilder"/> instance to add the SQL Server Database project to.</param>
+    /// <param name="name">Name of the resource.</param>
+    /// <returns>Am <see cref="IResourceBuilder{T}"/> that can be used to further customize the resource.</returns>
+    public static IResourceBuilder<SqlProjectResource> AddSqlPackage<TPackage>(this IDistributedApplicationBuilder builder, [ResourceName]string name)
```

Comment:
```suggestion
    public static IResourceBuilder<SqlProjectResource> AddSqlPackage<TPackage>(this IDistributedApplicationBuilder builder, [ResourceName] string name)
```

Reply:
Made the same change to the other methods as well.